### PR TITLE
Added service dashboard

### PR DESCRIPTION
**Pull request by thaingu**
This is still just a skeleton site with little functionality with regards to updating vacancies but is there any changes that should be made before functionality is added and the page is made dynamic?

### CODE REVIEW

**Comment by alexarnold94** [comment]
I'm unable to view the page; I get the error "Cannot read the property 'name' of undefined" when navigating to /service

File: app_server/views/layout.pug [hunk 1]
```diff
@@ -9,6 +9,7 @@ head
     link(href="https://fonts.googleapis.com/css?family=Roboto:300,400", rel="stylesheet")
     link(rel='stylesheet', href='/bootstrap/css/bootstrap.min.css')
     link(rel='stylesheet', href='/stylesheets/style.css')
+    script(src='https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js')
```
Comment (noobling) [inline app_server/views/layout.pug]: Hm im guessing you need jQuery but its already included at the bottom. I guess we could make a new issue to move all the script files into the `head`

File: app_server/views/service.pug [hunk 2]
```diff
@@ -1,43 +1,215 @@
...
+          .panel.panel-default
+            .panel-body
+              .col-sm-9
+                h4 Youth Accommodation Support Service
+                h5 Victoria Park
+              .col-sm-3
+                .pull-right
+                  h4
+                    | Available Beds:
+                    span.badge.badge-success 7
```
Comment (noobling) [inline app_server/views/service.pug]: Add some space here because there is no space `Available Beds:7`
You can do this in many ways preferably with some existing classes by bootstrap of simply add a space character or styles e.g. `padding-right:5px;`

File: app_server/views/service.pug [hunk 3]
```diff
@@ -1,43 +1,215 @@
...
-                .col-md-7.col-xs-12
-                    label(for="description")
-                        | Description
-                    textarea.form-control(type="text", rows=5, id="description", name="description")
-        .form-group
-            .text-right
-                input(type="submit", value="Submit").btn.btn-default
+          .col-xs-12
+            .thumbnail
+              .panel-body(style='height: 500px')
```
Comment (noobling) [inline app_server/views/service.pug]: If you were not aware inline styles is considered bad practice. Personally I don't mind this especially for our project.

**Comment by noobling** [comment]
> I'm unable to view the page; I get the error "Cannot read the property 'name' of undefined" when navigating to /service

You need to be logged in the access that page. We should really redirect the user and tell them they need to be logged in for that page.

File: app_server/views/service.pug (hunk 3 above)
Comment (thaingu) [reply app_server/views/service.pug]: I didn't know this before, I will change it for the future.

File: app_server/views/layout.pug (hunk 1 above)
Comment (thaingu) [reply app_server/views/layout.pug]: You're right I'll remove it

**Comment by thaingu** [comment]
Please review my code again, thanks.
